finquant/portfolio.py

```diff
@@ -648,6 +683,9 @@ def properties(self):
         string += f"\nPortfolio Expected Return: {self.expected_return:0.3f}"
         string += f"\nPortfolio Volatility: {self.volatility:0.3f}"
         string += f"\nPortfolio Sharpe Ratio: {self.sharpe:0.3f}"
+        string += f"\nPortfolio Value at Risk: {self.var:0.3f}"
+        string += f"\nConfidence level of Value at Risk: "
+        string += str(round(self.var_confidence_level * 100)) + "%"
```

Comment:
just minor correction: for sake of consistency, let's make this an f string as well:
```suggestion
        string += f"{str(round(self.var_confidence_level * 100))} %"
```